index.html

```diff
@@ -44,6 +44,158 @@ <h2 class="name1">I’am a Software Developer</h2>
+     <div class="tags">
+      <ul>
+        <li class="tag css">css</li>
+        <li class="tag html">html</li>
+        <li class="tag bootstrap">bootstrap</li>
+        <li class="tag ruby">Ruby</li>
+      </ul>
+      </div>
+      <button class="projects"><span class="see-projects">See project</span></button>
+  </div>
+   </div>
+     <div class="showcase">
+       <div class="Art">
+        <br><br><br>br
+           <h2 class="stories1"> Profesional Art Printing Data</h2>
+           <p class="paragraph1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         <div class="tags1">
+          <ul>
+            <li class="tag html1">html</li>
+            <li class="tag bootstrap1">bootstrap</li>
+            <li class="tag ruby1">Ruby</li>
+          </ul>
+         </div>
+           <button class="projects1"><span class="see-projects1">See project</span></button>
+      </div>
+     </div>
+     <div class="showcase">
+        <div class="Art">
+          <br><br><br>
+           <h2 class="stories1"> Profesional Art Printing Data</h2>
+           <p class="paragraph1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         <div class="tags1">
+           <ul>
+             <li class="tag html1">html</li>
+             <li class="tag bootstrap1">bootstrap</li>
+             <li class="tag ruby1">Ruby</li>
+           </ul>
+         </div>
+          <button class="projects1"><span class="see-projects1">See project</span></button>
+       </div>
+      </div>
+     <div class="showcase">
+       <div class="Art">
+        <br><br><br>
+         <h2 class="stories1"> Profesional Art Printing Data</h2>
+         <p class="paragraph1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+       <div class="tags1">
+         <ul>
+           <li class="tag html1">html</li>
+           <li class="tag bootstrap1">bootstrap</li>
+           <li class="tag ruby1">Ruby</li>
+         </ul>
+       </div>
+       <button class="projects1"><span class="see-projects1">See project</span></button>
+     </div>
+     </div>
+
+     <div class="showcase">
+       <div class="Art">
+        <br><br><br>
+         <h2 class="stories1"> Profesional Art Printing Data</h2>
+         <p class="paragraph1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         <br>
+       <div class="tags1">
+         <ul>
+           <li class="tag html1">html</li>
+           <li class="tag bootstrap1">bootstrap</li>
+           <li class="tag ruby1">Ruby</li>
+         </ul>
+       </div>
+         <br>
+         <button class="projects1"><span class="see-projects1">See project</span></button>
+     </div>
```

Comment:
- [ ] Here too, let's remove the span and give some styles to the button instead, as it will help us avoid unnecessary codes :recycle: